quickstarts/oma-aqm/config.yml

```diff
@@ -0,0 +1,44 @@
+# Name of the quickstart (required)
+name: alert-quality-management
+
+# Displayed in the UI (required)
+title: Alert Quality Management Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  Alert Quality Management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact.  This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+  
+  For more information, see https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Reduce the volume of nuisance incidents with the Alert Quality Management dashboard and process.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Alec Isaacson (New Relic)
+  - Andrew Faria (New Relic)
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - oma
+  - observabilty maturity
```

Comment:
is this an actual term that is worthy of a dedicated keyword?

Reply:
Yes.  OMA and "observability maturity" are terms that are part of an overall program created by NR to help our customers get better value from their observability solution.  See https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/introduction/ fro details